charts/openmetadata/templates/deployment.yaml

```
@@ -51,6 +51,19 @@ spec:
           readinessProbe:
             {{ .Values.readinessProbe | toYaml | indent 12 | trim }}
           env:
+          {{- if .Values.global.fernetkey.secretKey }}
```

Let's update this to say 
```suggestion
          {{- if .Values.global.fernetkey.secretRef }}
```